### ISSUE TITLE
Issue Convert Tensorflow 1.15 model failed

### ISSUE THREAD

**Issue by vonchenplus** [tf-coreml/tf-coreml#362]
## 🐞Describe the bug
When I trying to convert tensorflow 1.15 model to coreml model failed, 
I don't know if this is a problem with tensorflow or a problem with tfcoreml.

## Trace
```
Now finding ops in the TF graph that can be dropped for inference
Collecting all the 'Const' ops from the graph, by running it....
Traceback (most recent call last):
  File "/home/ateam/anaconda3/envs/coremlconvert/lib/python3.7/site-packages/tensorflow_core/python/client/session.py", line 1365, in _do_call
    return fn(*args)
  File "/home/ateam/anaconda3/envs/coremlconvert/lib/python3.7/site-packages/tensorflow_core/python/client/session.py", line 1350, in _run_fn
    target_list, run_metadata)
  File "/home/ateam/anaconda3/envs/coremlconvert/lib/python3.7/site-packages/tensorflow_core/python/client/session.py", line 1443, in _call_tf_sessionrun
    run_metadata)
tensorflow.python.framework.errors_impl.OutOfRangeError: Node 'mask/Conv/BatchNorm/FusedBatchNormV3' (type: 'Add', num of outputs: 1) does not have output 5
```

## System environment (please complete the following information):
 - coremltools version  (3.31):
 - tf-coreml version (1.1)
 - tensorflow version (1.15):
 - OS (Linux):
 - python version (3.7):

**Comment by Tigrex161** [comment]
I used my own training data to perform transfer learning on http://download.tensorflow.org/models/object_detection/ssdlite_mobilenet_v2_coco_2018_05_09.tar.gz
this pretrained ssdlite model. 

The model performs as expected after training. I only changed the number of classes to 1 in the pipeline.config file before training.

Converting the model to tflite format was also successful. Now i need it to run on iOS as well but when trying to convert(even after removing the excess parts of the graph using (https://github.com/apple/coremltools/blob/master/examples/neural_network_inference/tensorflow_converter/Tensorflow_1/ssd_example.ipynb) I also run into the mentioned error. 

I inspected the graph using netron and I'm attaching the pictures here. 
![network1](https://user-images.githubusercontent.com/13764071/70139460-5e492e00-16b4-11ea-863b-6ac40ae7dc99.PNG)
![network2](https://user-images.githubusercontent.com/13764071/70139473-63a67880-16b4-11ea-83c8-5ff9d26e865d.PNG)

The batchnorm node causing the error is "FeatureExtractor/MobilenetV2/Conv/BatchNorm/FusedBatchNormV3"
and it does indeed have just one output when inspected. Is it actually supposed to have 5 outputs? What am I doing wrong?
